pages/about/privacy-policy.vue

```diff
@@ -1,9 +1,36 @@
 <template>
-    <p>PRIVACY POLICY</p>
+    <div class="px-64 py-24">
+        <h1 class="text-3xl my-8 text-center">2021 {{ $t('title') }}</h1>
+        <div v-for="content in $t('contents')" :key="`${content}`" class="my-4">
+            <h1 class="text-2xl my-8 text-center">
+                {{ content.title }}
+            </h1>
+            <p
+                v-for="item in content.description"
+                :key="`${item}`"
+                class="m-2"
+                v-html="item"
+            />
+            <ol class="m-2">
+                <li v-for="(item, index) in content.items" :key="`${item}`">
+                    {{ index + 1 }}. {{ item }}
```

Comment:
If you need the index of items to be displayed for the ordered list, please [change the style type of the component](https://tailwindcss.com/docs/list-style-type#usage) instead of inserting it manually.